src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -47,6 +47,8 @@
      */
     public static final String DEFAULT_NAME = "Gencode";
 
+    public static final String OTHER_TRANSCRIPTS_INFO_SEP = "_";
```

Comment:
I think this has been  defined elsewhere....  As a "/" maybe?

Reply:
This was meant to capture the `_` character that separates the pieces of information in the `OtherTranscripts` field. 

@LeeTL1220 Are you suggesting to rename the variable?